tests/lang/s01_basics/s08_user_defined_data_types/t02_plain_objects_fail1_illegal_recursion.nim

```diff
@@ -10,29 +10,31 @@ illegal recursion in type 'Object'
 """
 
 
-## It is possible to use object recursively in itself, but only 
+## It is possible for an object to contain itself recursively, but only 
 ## if it was defined or used as a `ref`/`ptr`
 ## 
 ## Regular objects are embedded directly in the resulting structure, so
-## if object was to be 'embedded' in itself, it would have an infinite size.
+## if an object was to be 'embedded' in itself, it would have an infinite size.
 ## Using `ptr/ref` solves this issue - they have a fixed size.
 
 block with_ref_object:
   ## No error here because using `ref object`
   type
     Object = ref object
       field: Object
+      ## Ref is 8 byte referenced pointer; therefore object is 8 bytes in size
```

Comment:
Assert size with `assert sizeof() == ` if it hasn't been done already elsewhere.